src/bandersnatch_storage_plugins/swift.py

```diff
@@ -720,6 +719,21 @@ def copy_file(
             conn.copy_object(self.default_container, str(source), dest)
         return
 
+    def move_file(
+        self, source: PATH_TYPES, dest: PATH_TYPES, dest_container: Optional[str] = None
+    ) -> None:
+        """Move a file from **source** to **dest**"""
+        if dest_container is None:
+            dest_container = self.default_container
+        dest = f"{dest_container}/{dest}"
+        with self.connection() as conn:
+            conn.copy_object(self.default_container, str(source), dest)
+            try:
+                conn.delete_object(self.default_container, str(source))
+            except swiftclient.exceptions.ClientException:
+                raise FileNotFoundError(str(source))
```

Comment:
What's the advantage of raising `FileNotFoundError` rather than just letting the swift exception raise?

Reply:
This is just copied from SwiftStorage.delete_file(), I don't have an opinion on which is better, ;-)